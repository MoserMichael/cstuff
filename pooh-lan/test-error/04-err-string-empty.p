

r = '

# this was an error
