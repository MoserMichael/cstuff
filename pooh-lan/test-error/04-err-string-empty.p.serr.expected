
test-error/04-err-string-empty.p(4): Error: The string constant is not closed, it goes on and on until the end of the file
      4 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |
        |^

test-error/04-err-string-empty.p(3): Error: syntax error, unexpected error 
      3 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |r = '
        |....^

test-error/04-err-string-empty.p(3): Error: in assignment; right hand side (value of assignment) is wrong
      3 |:....+....:....+....:....+....:....+....:....+....:....+....:....+....:....+....
        |r = '
        |....^
Parsing failed
